tasmota/xsns_53_sml.ino

```diff
@@ -2273,6 +2275,10 @@ void SML_Init(void) {
     dvalid[cnt]=0;
   }
 
+  for (uint32_t cnt=0;cnt<MAX_DVARS;cnt++) {
+    dvalues[cnt]=0;
```

Comment:
Why not using 'memset()' on the whole array instead of a loop?

Reply:
sure memset() could be used; I just followed what was already written